packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -440,10 +445,9 @@ private void prepareMediaRecorder(String outputFilePath) throws IOException {
       mediaRecorder.setVideoSize(videoSize.getWidth(), videoSize.getHeight());
       mediaRecorder.setOutputFile(outputFilePath);
 
-      int displayRotation = activity.getWindowManager().getDefaultDisplay().getRotation();
-      int displayOrientation = ORIENTATIONS.get(displayRotation);
-      if (isFrontFacing) displayOrientation = -displayOrientation;
-      mediaRecorder.setOrientationHint((displayOrientation + sensorOrientation) % 360);
```

Comment:
Are the other places where displayOrientation is used still correct?

Reply:
No, updated. Good catch, thanks.